OracleWebLogic/dockerfiles/14.1.1.0/Dockerfile.developer

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020, Oracle Corporation and/or its affiliates. 
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for WebLogic 14.1.1.0 Quick Install Distro
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_14.1.1.0.0_wls_quick_Disk1_1of1.zip
+#     Download the Developer Quick installer from http://www.oracle.com/technetwork/middleware/weblogic/downloads/wls-for-dev-1703574.html
+#
+# (2) jdk-11.0.6_linux-x64_bin.tar.gz 
```

Comment:
Is this actually necessary for this image? It's required to build `oracle/jdk:11` but not for this image specifically.

Reply:
This image extends the jdk 11 image so I would think it is required.
What do you mean by "but not or this image specifically" ?

Reply:
I mean that the documentation for building the JDK11 would have this dependency, but this documentation doesn't need it. For example, you could use the JDK11 image from the Container Registry instead.